src/mode/modeNormal.ts

```diff
@@ -26,7 +26,7 @@ export default class NormalMode extends Mode {
         "dd" : () => { return vscode.commands.executeCommand("editor.action.deleteLines"); },
         "dw" : () => { return vscode.commands.executeCommand("deleteWordRight"); },
         "db" : () => { return vscode.commands.executeCommand("deleteWordLeft"); },
-        // "x" : () => { this.CommandDelete(1); }
+        "x" : () => { return vscode.commands.executeCommand("deleteRight"); },

```

Comment:
Yeah, I expect that to happen, but I need to have small things like x, or cw available in the meantime. Also I'll probably try to add some dumb visual selection. I don't use all of what vim offers, mostly stuff I got used to.

I can deploy manually for myself if you think it's better to wait for the proper handling implementation.